tests/builders/test_table_builder.py

```diff
@@ -117,7 +117,11 @@ def create_tables(self):
                     #       so we just try to create if fails we continue
                     raise
         finally:
-            self.connection.commit()
+            try:
+                self.connection.commit()
+            except AttributeError: # sqla < 1.4
```

Comment:
This comment is confusing.
The first line says sqla<1.4
But second line talks about sqla<2.0

Could you clearly explain what happens in which version